contrib/lkt/extensions/mains/lkt_toolbox.adb

```diff
@@ -90,6 +90,12 @@ begin
             Unit : constant Analysis.Analysis_Unit
               := Ctx.Get_From_File (To_String (File_Name));
          begin
+            --  We explicitly call Populate_Lexical_Env here in order to be
+            --  able to print the diagnostics about invalid import statements
+            --  (U914-006). Indeed, waiting the call to P_Check_Semantics below
```

Comment:
Except for temporary TODOs, I would suggest not to leave ticket numbers in comments: the codebase is public whereas tickets aren’t, so permanent comments/doc/… should be self-sufficient.

Reply:
I see your point. I'll fix that!